change/msal-b56fcb92-f186-43cd-be5f-ed0834c67b12.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add AzureCloudInstance support for JS libraries(#4106)",
```

```suggestion
  "comment": "Add AzureCloudInstance support for JS libraries #4106",
```

(for each of the change files)

---

And the only change for `msal-core` in this PR looks to be the `package.json` which can be reverted, so this specific change file can be removed.

---

Agree